### DISCUSSION TITLE
How to add reputation in debug console in game

### DISCUSSION THREAD

**Discussion by kk23abc** [Vdauphin/HeartsAndMinds#1415] [Question and answer]
Hi there, may I know what to put on debug console to add or minus reputation in debug console in game? TQ

**Comment by Vdauphin** [comment]
Hello,
Exec server side: `[-10] call btc_rep_fnc_change;`
See https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/rep/change.sqf
Cheers

**Reply by kk23abc** [reply]
Thanks man..
